### DISCUSSION TITLE
Full, full width w/ stage manager

### DISCUSSION THREAD

**Discussion by nfiniteset** [rxhanson/RectanglePro-Community#230] [Ideas]
I love that Rectangle reserves space for the Stage Manager window previews. It works great when I want a two-up of browser windows, zoom, calendar, email client, etc.

But for my content-oriented work apps like Figma or AfterEffects, I like to go truly full-width so I can see maximum content. In this case I have to resort to dragging the window wider with my mouse. 

I wonder how Rectangle could honor the window previews but also let me go truly full-width via keystroke.

One idea is for the fullscreen keystroke to honor window-previews on the first press, then on a second press ignore window-previews and go full width.

**Comment by rxhanson** [comment]
Thanks for the feedback. I had to fix a bug I introduced in the version yesterday, so I pushed a quick change in for you in 2.7.9 (update within the app or at https://rectangleapp.com/pro).

Once you're on 2.7.9, execute this terminal command to get the behavior you describe:

defaults write com.knollsoft.Hookshot repeatedMaximizeMode -int 5

Then restart the app.

**Reply by nfiniteset** [reply]
It works great. Thanks!

**Reply by nonameolsson** [reply]
Excellent, just what I was looking for! 

Does this exist somewhere as a setting in the UI?

**Reply by rxhanson** [reply]
It's not in the UI, but I might add it in a future release.

**Comment by nfiniteset** [comment]
Been using this daily since you released it for me about a year ago. Thanks again! 🙏
